### DISCUSSION TITLE
Prevent conditional instructions like IF from suffering from the "Magic Number" anti-pattern

### DISCUSSION THREAD

**Discussion by MarqReal** [LemonAppDev/konsist#1162] [Request a Feature]
### Tell us more.

Hello everyone, everything good?

Firstly I want to thank you for the project, it is fantastic!!

I was wondering if it would be possible to have a feature that validates whether a conditional instruction has a **[magic number](https://medium.com/@samaddico/programming-and-magic-numbers-f766e0cd1369)**

Forgive me if this has already been mapped or is already present in the lint, I didn't see it :/

I would like to make myself available if there is an opportunity to carry out an inner source or participate in the construction of this feature or others.

**Comment by chrys-aa** [comment]
I could really make usage of this feature @MarqReal!! Great idea!

**Reply by igorwojda** [reply]
ATM Konsist supports high level declarations modules/classes/interfaces/, but no function bodies. 
We may consider this in the future, but it may be quite challenging.
